src/explorer/trialApp/TrialAppTreeItemBase.ts

```diff
@@ -0,0 +1,44 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzExtParentTreeItem, AzExtTreeItem, TreeItemIconPath } from 'vscode-azureextensionui';
+import { localize } from '../../localize';
+import { getIconPath } from '../../utils/pathUtils';
+import { ITrialAppMetadata } from './ITrialAppMetadata';
+
+export abstract class TrialAppTreeItemBase extends AzExtTreeItem {
```

Comment:
I think you should have a SiteTreeItemBase that are both extend by SiteTreeItem and TrialAppTreeItem.  TrialAppTreeItemBase is only used to extend TrialAppTreeItem.

I think since you've been making the interfaces in the tools package, you probably have a better idea of what I'm trying to get you to do, but let me know if you need any clarification?

Reply:
Makes sense. However, I think I need to wait until we publish the tools package to make this change. Once it's published, it will allow SiteTreeItem to no longer extend AzureParentTreeItem.